plugin-admin/generate_plugins.js

```diff
@@ -0,0 +1,101 @@
+// Create empty object structure
+// Read plugins.ejs object in here and save as local variable
+// For each dep in package.json (as we don't want the ones not on npm):
+//   NOTE: scoped packages under node_modules/@seneca/title, unscoped under node_modules/name
+//   Write relevant information from package.json (stored in node_modules folder) to object
+//   Write relevant information not found in package.json from plugins.ejs variable to object
+//   NOTE: how to get groups if not every plugin.ejs is going in?
+// End for
+// Convert data to string to send
+// Write string data to file in /plugins-2023 to be included in another file in that dir
+const fs = require('fs')
+const jsonic = require('jsonic')
+
+let pluginData = {}
+
+let pkgjson = require('./package.json')
+Object.keys(pkgjson.dependencies).forEach((dep) => {
+  // npm fields: name, title, org_repo, desc
+  let pkg = require('./node_modules/' + dep + '/package.json')
```

Comment:
perform this as a separate step, using reduce to create a keyed map from dep key to pkg json - this is more flexible

```
const pkgmap = ...dependencies).reduce(
  ( (pkgmap, dep)=>(pkgmap[key]=dep, pkgmap) ),
  {}
)
```